libraries/libpensol/src/test/java/org/pentaho/reporting/libraries/pensol/JCRSolutionFileModelTest.java

```diff
@@ -29,8 +29,8 @@
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.*;
```

Comment:
Expand this to just the classes being used.

Reply:
expanded.